tests/test.yargs-options.js

```diff
@@ -4,6 +4,7 @@ describe('getDefaultConfigValue()', () => {
   it('should return the default value', () => {
     expect(getDefaultConfigValue('self-hosted')).toEqual(false);
     expect(getDefaultConfigValue('privileged')).toEqual(false);
+    expect(getDefaultConfigValue('max-manifest-version')).toEqual(3);
```

Comment:
There were only 'self-hosted' and 'privileged' tests.
Should we add more tests like 'min_manivest_version' and so on?